test/ci-cd/cloudbuild/platforms-gke-base-core-terraform.yaml

```diff
@@ -0,0 +1,53 @@
+options:
+  logging: CLOUD_LOGGING_ONLY
+
+steps:
+- args:
+  - --cache=true
+  - --cache-ttl=48h
+  - --context=dir://test/ci-cd/container_images
+  - --destination=${_AR_REPO_LOCATION}-docker.pkg.dev/${PROJECT_ID}/ci-cd/runner:latest
+  - --dockerfile=test/ci-cd/container_images/dockerfile.runner
+  - --log-format=text
+  - --log-timestamp=false
+  - --verbosity=info
+  id: "Build runner image"
+  name: "gcr.io/kaniko-project/executor:latest"
+  waitFor: ["-"]
+
+- name: "${_AR_REPO_LOCATION}-docker.pkg.dev/${PROJECT_ID}/ci-cd/runner:latest"
+  id: "Test platforms/gke-aiml/playground Terraform"
+  entrypoint: "ash"
+  args:
+    - "-x"
+    - "-c"
+    - |
+      export NO_COLOR="1"
+      export TERM="xterm"
+      export TF_CLI_ARGS="-no-color"
+      export TF_IN_AUTOMATION="1"
+
+      export ACP_REPO_DIR="/workspace"
+      export ACP_PLATFORM_BASE_DIR="/workspace/platforms/gke/base"
+      export ACP_PLATFORM_CORE_DIR="/workspace/platforms/gke/base/core"
+
+      export TF_VAR_cluster_project_id="${PROJECT_ID}"
+      export TF_VAR_platform_name="commit-${SHORT_SHA}"
+      export TF_VAR_terraform_project_id="${PROJECT_ID}"
+
+      platforms/gke/base/core/deploy.sh
+      platforms/gke/base/core/deploy.sh
+
+      APPLY_RETURN_CODE=$?
+      echo "APPLY_RETURN_CODE=$${APPLY_RETURN_CODE}"
+
+      platforms/gke/base/core/teardown.sh || exit 1
+      if [[ $${APPLY_RETURN_CODE} -ne 0 ]]; then
+        exit $${APPLY_RETURN_CODE}
+      fi
+  waitFor:
+  - "Build runner image"
+
+substitutions:
+  _AR_REPO_LOCATION: "us-central1"
+  _IAP_DOMAIN: "accelerated-platforms.joonix.net"
```

Comment:
Maybe tell people to substitute with their own domain. Is there somewhere an explanation on how to setup IAP for this solution ?

Reply:
So these are currently only used for our internal CI/CD on the repo. If we decide to make a version for public consumption I will create a README for it.